### DISCUSSION TITLE
申请退款！！

### DISCUSSION THREAD

**Discussion by Elder-Jiang** [getlantern/lantern-forum-cn#1009] [General]
刚刚开通的pro，根本用不了，麻烦退一下款，谢谢
账号：jiangyichong@outlook.com
![微信图片_20241030210133](https://github.com/user-attachments/assets/5794553e-dc30-46d3-a408-fe4c9ad863f3)

**Comment by MaxMKMax** [comment]
@Elder-Jiang 其实我们正在修复了. 如果您不介意的话等我们一周. 您就应该可以用了. 
如果您的问题是收不到激活码或者授权码. 我们这里可以帮到您. 这样您就可以使用了.
请问您无法使用的原因是什么?
